packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/templates/expandableSearchBoxInput.css

```diff
@@ -37,6 +37,7 @@ textarea.searchbox__input {
   border: none;
   box-shadow: none;
   overflow-x: clip;
+  overflow-y: hidden;
```

Comment:
it starts hidden but the searchbox can still expand when you click in it right?

Reply:
indeed yeah!